docs/ibc/proto-docs.md

```diff
@@ -3585,10 +3583,7 @@ WASM light client's Client state
 | ----- | ---- | ----- | ----------- |
 | `data` | [bytes](#bytes) |  |  |
 | `code_id` | [bytes](#bytes) |  |  |
-| `frozen` | [bool](#bool) |  |  |
```

Comment:
this isn't required anymore since `IsFrozen` method has been replaced with `Status`